docs/modules/ROOT/pages/how-tos/testing/integration/creatingoverridesnapshot.adoc

```diff
@@ -30,19 +30,22 @@ spec:
   components:
     - name: component-sample <2>
       containerImage: quay.io/redhat-user-workloads/ws-sample-tenant/application-sample/component-sample@sha256:0db0a473a6abf5c15c424ab07cfbd5c40c06622fe648d4fe6a6b6abc224a0d0c <3>
+      source: <4>
+        git:
+          revision: fa8b89274a61ef0f1c257b7a84c37aa2ec844109
 ----
 <1> The label `test.appstudio.openshift.io/type: override` that indicates this is an `override` snapshot.
 <2> The component name you will reset its Global Candidate List.
 <3> The container image with valid digest you will reset the component <2> to.
-
+<4> Optional. The component source.
```

Comment:
Maybe we can expand this to say `Optional. The component source containing the git revision.`

Reply:
Updated. Thanks